l10n/en/firefox/browsers/windows-64-bit.ftl

```diff
@@ -0,0 +1,39 @@
+
+### URL: https://www-dev.allizom.org/firefox/browsers/windows-64-bit/
+
+
+# HTML page title
+windows-64-bit-firefox-for-windows = { -brand-name-firefox } for { -brand-name-windows } 64-bit
+
+# HTML page description
+windows-64-bit-users-on-64-bit-windows = Users on 64-bit { -brand-name-windows } who download { -brand-name-firefox } can get our 64-bit version by default. That means you get a more secure version of { -brand-name-firefox }.
+windows-64-bit-64-bit = 64-bit
+
+# Main title
+windows-64-bit-a-more-secure-firefox = A more secure { -brand-name-firefox }.
+
+# Variables:
+# $crashes (url) - link to https://blog.mozilla.org/firefox/defeat-browser-crashes/
+windows-64-bit-users-on-64-bit-windows-crashes = Users on 64-bit { -brand-name-windows } who download { -brand-name-firefox } can get our 64-bit version by default. That means you get a more secure version of { -brand-name-firefox }, one that also <a href="{ $crashes }">crashes a whole lot less</a>. How much less? In our tests so far, 64-bit { -brand-name-firefox } reduced crashes by 39% on machines with 4GB of RAM or more.
+windows-64-bit-whats-the-difference = What’s the difference between 32-bit and 64-bit?
+
+# Variables:
+# $ASLR (url) - link to https://en.wikipedia.org/wiki/Address_space_layout_randomization
+# The term "fret not" can be translated as "not to worry", or "don’t worry".
+windows-64-bit-heres-the-key-thing = Here’s the key thing to know: 64-bit applications can access more memory and are less likely to crash than 32-bit applications. Also, with the jump from 32 to 64 bits, a security feature called <a href="{ $ASLR }">Address Space Layout Randomization (ASLR)</a> works better to protect you from attackers. { -brand-name-linux } and { -brand-name-mac } users, fret not, you already enjoy a { -brand-name-firefox } that’s optimized for 64-bit.
+windows-64-bit-how-do-you-get-64 = How do you get 64-bit { -brand-name-firefox }?
+
+# Variables:
+# $version (url) - link to https://support.microsoft.com/help/13443/windows-which-operating-system
+# $check (url) - link to https://support.mozilla.org/kb/update-firefox-latest-version
+windows-64-bit-if-youre-running = If you’re running 64-bit { -brand-name-windows } (<a href="{ $version }">here’s how to check</a>), your { -brand-name-firefox } may already be 64-bit. <a href="{ $check }">Check your { -brand-name-firefox } version</a> (in the “About { -brand-name-firefox }” window) and look for “(32-bit)” or “(64-bit)” after the version number:
+windows-64-bit-if-you-see-32-bit-older = If you see “(32-bit)” and you are running { -brand-name-firefox } 56.0 or older, updating to the latest { -brand-name-firefox } version should automatically upgrade you to 64-bit.
+windows-64-bit-if-you-see-32-bit-newer = If you see “(32-bit)” and are running Firefox 56.0.1 or newer, then your computer may not meet the minimum memory requirement for 64-bit (3 GB RAM or more). You can still manually install 64-bit Firefox, if you choose.
```

Comment:
@craigcook missed `Firefox` in this string. Please create another PR to fix this.